doc/man1/openssl-smime.pod

```diff
@@ -332,7 +331,12 @@ B<-auth_level>, B<-verify_depth>, B<-verify_email>, B<-verify_hostname>,
 B<-verify_ip>, B<-verify_name>, B<-x509_strict>
 
 Set various options of certificate chain verification. See
-L<verify(1)> manual page for details.
+L<openssl-verify(1)> manual page for details.
+
+=item I<cert.pem> ...
+
+One or more certificates of message recipients: used when encrypting
```

Comment:
Why the colon : ?

Reply:
I have *no* clue

Reply:
Ah, that wasn't me!  Plus, I forgot to remove the B<cert.pem> item 20 lines up...

Reply:
it should be a comma, but can be a semicolon if you want to be pretentious. :)

Reply:
It is a comma now